### PR TITLE
Fix iframe modal scrollbars

### LINKED ISSUE #25735: Product Page v2: Fix the scrollbars on the iframe for attachment form
#### Describe the bug

Previously fixed with this PR: https://github.com/PrestaShop/PrestaShop/pull/26024
In product page v2 when adding a new file a modal opens containing an iframe that includes the form.
This modal has unwanted scrollbars, we need to find a way to remove those (maybe the max width is too
small, maybe the auto size feature is lacking a few pixels)

#### Expected behavior

No scroll bars

#### Steps to Reproduce

1. Go to Product page V2
2. Go to Specifications tab 
3. Click on 'Add new file'
4. See the unwanted scrollbars (see screenshot)

**Screenshots**

Capture d’écran 2021-08-30 à 11 43 57

#### Additional information

* PrestaShop version: 8.0
* PHP version: N/A

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Fix scrollbars bug in IframeModal component
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #25735
| How to test?      | Test that the iframe modal has no scrollbars anymore, mostly used on new product page (creation prudtc modal, attachment creation modal)
| Possible impacts? | ~

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/28050)

### CODE REVIEW

File: admin-dev/themes/new-theme/js/components/modal/iframe-modal.ts [hunk 1]
```diff
@@ -145,13 +151,20 @@ export class IframeModal extends Modal implements IframeModalType {
...
-        this.autoResize();
+        this.initAutoResize();
       }
     });
 
     this.$modal.on('shown.bs.modal', () => {
       this.modal.iframe.src = params.iframeUrl;
     });
+
+    // @ts-ignore
```
Comment (NeOMakinG) [inline admin-dev/themes/new-theme/js/components/modal/iframe-modal.ts]: Wut? Why?

**Comment by NeOMakinG** [comment]
@jolelievre could we plan a 1:1 about this? I'm not really understanding the need for this layer over HTML Events, I may miss something

Also, your `@ts-ignore` made me think that we should really dive into every `@ts-ignore` in the project, pretty sure most of these can be removed now

**Review by NeOMakinG — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Review by florine2623 — CHANGES_REQUESTED** [review]
Hello @jolelievre ,

Thanks for the PR !

I have a scrollbar when I zoom to 200% or more on my browser :

https://user-images.githubusercontent.com/16019289/160616065-ccab1ceb-6a1b-4240-b4d1-24aa81dde042.mov

Also, I have a black border on that modal. I don't think it should be here as shown in this comment : https://github.com/PrestaShop/PrestaShop/pull/26024#issuecomment-998143843 :
Screenshot 2022-03-29 at 14 59 03

Could you check ? 

Otherwise it looks good to me ! I checked on desktop as well as mobile. Files are well uploaded.

| Screenshot 2022-03-29 at 14 53 33 | Screenshot 2022-03-29 at 14 53 57 |
| ------------- | ------------- |

Thanks!

**Comment by jolelievre** [comment]
Hi @florine2623 

about the border it's not related to this PR As far as I can see it is from the boostrap scss https://github.com/twbs/bootstrap/blob/dca1ab7d877bc4b664b43604657a2b5fbe2b4ecb/scss/_modal.scss#L116 so it has always been there.

About the zoom, I tested it but I couldn't reproduce your issue. I'm not sure it's part of the PR either seems like an edge case I didn't know we were supposed to support all custom modifications made by the user on its own environment.

To confirm with @MatShir

**Comment by MatShir** [comment]
The dark borderline wasn't introduced by this PR, and can be found it other modal in the back office in the develop branch. For this part, I suggest making a dedicated issue with the scroll bar behavior. So we can discuss it with @TristanLDD.

**Comment by MatShir** [comment]
If it is the last bug from this PR, is the PR ready ? 😄

**Review by florine2623 — APPROVED** [review]
Hello @jolelievre & @MatShir ,

Thanks for your comments !
Then it looks good to me ^^
It is QA ✅

**Pull request merged**